css/layout.css

```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans:wght@400;700&display=swap');
+
+body {
+  color: #925d3b;
+  font-family: 'Noto Sans', sans-serif;
+}
+
+.hidden {
+  display: none !important;
```

Comment:
instead of using important here, I would recommend putting this class at the bottom of the CSS document but just above the media queries. It is better to use the cascading nature of CSS to make sure this class is applied rather than important.